draft-ietf-quic-applicability.md

```diff
@@ -379,10 +380,10 @@ application on top.
 
 Priority handling of retransmissions can be implemented by the sender in the
 transport layer. {{QUIC}} recommends retransmitting lost data before new data,
-unless indicated differently by the application. Currently, QUIC only provides
-fully reliable stream transmission, which means that prioritization of
-retransmissions will be beneficial in most cases, by filling in gaps and freeing
-up the flow control window. For partially reliable or unreliable streams,
+unless indicated differently by the application. When a QUIC endpoint uses
+fully reliable streams for transmission, prioritization of retransmissions will
+be beneficial in most cases, as filling in gaps and freeing up the flow
```

Comment:
I prefer either "by" as you had it before or drop "as" entirely.

Reply:
```suggestion
be beneficial in most cases, filling in gaps and freeing up the flow
```